packages/ui/styles/mixins.less

```diff
@@ -345,8 +348,89 @@
 	});
 }
 
+//
+// Locale Mixins
+//
+
+// .enact-locale-line-height()
+//
+// Assign line-height rules specifically to the languages designated
+// as needing special support for tall-glyphs.
+//
+// Set line-height for normal and tallglyphs with 1 list argument
+// Ex:
+//   .enact-locale-line-height(1.4em 1.6em);
+// Or:
+//   @lineheight: 1.4em 1.6em;
+//   .enact-locale-line-height(@lineheight);
+.enact-locale-line-height(@both) when (length(@both) = 2) {
+	.enact-locale-tallglyph(line-height, @both);
+}
+
+
+// .enact-locale-tallglyph()
+//
+// Rulename and a normal value followed by a tallglyph value (3 arguments)
+// Ex:
+//   .enact-locale-tallglyph(font-size; 1.4em; 1.6em);  ->  normal font-size: 1.4em; tallglyphs font-size: 1.6em;
+.enact-locale-tallglyph(@rule; @normal; @tallglyph) {
+	@{rule}: @normal;
+	.enact-locale-tallglyph(@rule, @tallglyph);
+}
+
+// Rulename and 2 value second variable (2 arguments, 2nd being a list)
+// Ex:
+//   .enact-locale-tallglyph(font-size; 1.4em 1.6em);  ->  normal font-size: 1.4em; tallglyphs font-size: 1.6em;
+// Or:
+//   @fontsize: 1.4em 1.6em;
+//   .enact-locale-tallglyph(font-size, @fontsize);    ->  normal font-size: 1.4em; tallglyphs font-size: 1.6em;
+.enact-locale-tallglyph(@rule; @val) when (length(@val) = 2) {
+	@{rule}: extract(@val, 1);
+	.enact-locale-tallglyph(@rule, extract(@val, 2));
+}
+
+// Rulename and a tallglyph value
+// Ex:
+//   .enact-locale-tallglyph(font-size; 1.6em);  ->  tallglyphs font-size: 1.6em;
+.enact-locale-tallglyph(@rule; @val) when (default()) {
+	.enact-locale-tallglyph({
+		@{rule}: @val;
+	});
+}
+
+// Accept a ruleset and apply it to the list of tall-glyph languages
+// Ex:
+//   .enact-locale-tallglyph({
+//     font-size: 1.6em;
+//   });
+.enact-locale-tallglyph(@rules) when (isruleset(@rules)) {
+	// :global(.enact-locale-th) &,
+	// :global(.enact-locale-si) & {
+	// 	@rules();
+	// }
```

Comment:
Going to remove these

Reply:
I had actually left this here on purpose, since this (commented out) approach is more verbose and less configurable on the development side, but it produces slightly smaller, more compact output code. Eg:
Commented out version:
```css
.enact-locale-si .BodyText_bodyText__1Fj1C,
.enact-locale-th .BodyText_bodyText__1Fj1C,
.enact-locale-vi .BodyText_bodyText__1Fj1C {
    line-height: 1.7em;
}
```
vs Current:
```css
.enact-locale-si .BodyText_bodyText__1Fj1C {
    line-height: 1.7em;
}
.enact-locale-th .BodyText_bodyText__1Fj1C {
    line-height: 1.7em;
}
.enact-locale-vi .BodyText_bodyText__1Fj1C {
    line-height: 1.7em;
}
```
This more verbose version that produces more compact output code could be desirable for efficiency and I think is possible to automate the generation of, but I just couldn't figure it out at this time.